docs/transparent-validation/sonoff/quick-start-guide.md

```diff
@@ -0,0 +1,160 @@
+# Sonoff documentation
+
+## Quick start guide
+
+The below documentation is based on the
+[EPSHome documentation](https://esphome.io/devices/sonoff_s20.html)
+
+1. Open the Sonoff case.
+
+1. Check if the headers are soldered to the board. If not - 4 pin header
+    raster 2.54 mm should be soldered to the board in accordance with the below
+    images.
+
+    ![sonoff_without_header](images/sonoff_without_header.jpg)
+
+    ![sonoff_with_header](images/sonoff_with_header.jpg)
+
+1. Connect the USB-UART converter to the USB port.
+
+    ![uart connection](images/sonoff-s20-uart.jpg)
+
+1. Create a Sonoff configuration file. The file extension should be `.yaml` and
```

Comment:
In what location this file should be placed?

Reply:
Any folder as far as I understand but I didn't created this part of documentation

Reply:
Updated